packages/svelte/src/internal/client/runtime.js

```diff
@@ -1253,22 +1253,33 @@ export function unwrap(value) {
 }
 
 if (DEV) {
-	/** @param {string} rune */
-	function throw_rune_error(rune) {
+	/**
+	 * @param {string} rune
+	 * @param {string[]} [variants]
+	 */
+	function throw_rune_error(rune, variants = []) {
 		if (!(rune in globalThis)) {
+			// TODO if people start adjusting the "this can contain runes" config through v-p-s more, adjust this message
 			// @ts-ignore
 			globalThis[rune] = () => {
-				// TODO if people start adjusting the "this can contain runes" config through v-p-s more, adjust this message
-				throw new Error(`${rune} is only available inside .svelte and .svelte.js/ts files`);
+				throw new Error(`${rune}() is only available inside .svelte and .svelte.js/ts files`);
```

Comment:
unrelated to this PR, but I wonder if we should prevent _accessing_ the rune, rather than adding a function to the global scope? right now things like `typeof $state === 'function'` will behave weirdly; i kinda feel like we should make them behave more like keywords, since that's closer to what they are. we can't throw a `SyntaxError` (which is what happens with things like `typeof return` but we can make `typeof $state` result in an error if we use `Object.defineProperty(window, ...)`